docs2/site/docs/migrations/migration5.md

```diff
@@ -150,6 +150,63 @@ and data loader methods are supported such as shown in the below example:
 The above example assumes that the `Person` CLR type was mapped to `PersonGraphType`
 in the schema CLR type mappings.
 
+Arguments are also added as query arguments to the field, and recognize default values
```

Comment:
I still do not like the term `query arguments`. I hope we will eventually change it.

Reply:
Fine but for now it is consistent, and helps to distinguishing from method arguments.